starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,52 @@
 // Names and Input
-
+var hacker1 = "Ryan";
+var hacker2 = "Diego";
```

Comment:
in order to ask user for an input your would have to use prompt   var hacker2 = prompt("what is your name?")